### ISSUE TITLE
DNS record mgmt accessibility check

### ISSUE THREAD

**Issue by witha-c** [cisagov/manage.get.gov#3817]
### Sub-issue description

After basic designs are completed, we need to review and plan for accessibility considerations. 

### Acceptance criteria

- [x] Trusted tester reviews current [wireframes ](https://figma-gov.com/design/4vp4ytRRMiULE2r7MLnZpo/.gov-registar?node-id=18931-13187&p=f&t=NjftjlJozKSQYsY3-0)for accessibility considerations. 
- [x] Pair with @witha-c to plan for annotations 

### Additional context

The goal for this is not to finalize accessibility for these designs, but rather to know what considerations are in play as we iterate on the designs and get feedback. This may yield some process improvements to help us with accessibility as we continue to build this feature.

**Comment by witha-c** [comment]
Sprinting into the future to note that it should be completed before the end of the current bridge POP

**Comment by witha-c** [comment]
Moving to 74 to make room for @SamiyahKey to tackle #3841 
cc @katypies

**Comment by SamiyahKey** [comment]
I am currently thinking through formalizing a repeatable a11y consideration process that will be part of the AC's for all design tickets. Will post here / discuss with Chrisina when she's back

**Comment by SamiyahKey** [comment]
Here is a new process discussed with @witha-c ! Designers will annotate buttons, header levels, and links as part of their a11y 
annotations. This will be incorporated into design AC's in prep for dev handoff. 

[Wireframe accessibility annotation guide](https://docs.google.com/document/d/1MIxfNyYj3f6QJ7gkzYbF9QtGj00jgDKZH9CN2cEdz9U/edit?usp=sharing)

**Comment by SamiyahKey** [comment]
small note to annotate that the TDL disabled field should be marked up using [ aria-disabled="true ]
